common/src/main/java/com/linecorp/centraldogma/internal/MismatchedValueException.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2018 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.centraldogma.internal;
+
+/**
+ * A {@link RuntimeException} that is raised when merging JSON files is failed due to mismatched value.
+ */
+public final class MismatchedValueException extends RuntimeException {
```

Comment:
How about more generic name like `MergeConflictException` or `EntryMergeConflictException`, because otherwise we will have to create an exception for every case where merge operation fails?

Reply:
Alternatively, maybe we could just use `QueryExecutionException` and do not add a new exception at all?

Reply:
Just used `QueryExecutionException`.